packages/visual-editor/src/index.ts

```diff
@@ -944,9 +944,10 @@ export class Main extends LitElement {
           this.settingsHelper
         );
         // Once we've determined the sign-in status, relay it to an embedder.
-        this.#embedHandler?.sendToEmbedder(
-          {type: "home_loaded", isSignedIn: signInAdapter.state === "valid"}
-        )
+        this.#embedHandler?.sendToEmbedder({
```

Comment:
we need to tell James to start using Prettier 😆

Reply:
This seems like a job for presubmit:)